File: FitnessWeb.API/Commands/CreateFitnessTrainingCommand.cs
using MediatR;

namespace FitnessWeb.API.Commands
{
    public class CreateFitnessTrainingCommand : IRequest<int>
    {
        public int FitnessProgramId { get; set; }
        public string Type { get; set; }
        public int Duration { get; set; }
    }
}

File: FitnessWeb.Frontend/AngularApp/src/commands/createFitnessTrainingCommand.ts
export class CreateFitnessTrainingCommand {
    type: string;
    duration: number;
    fitnessProgramId: number;

    constructor(type: string, duration: number, fitnessProgramId: number) {
        this.type = type;
        this.duration = duration;
        this.fitnessProgramId = fitnessProgramId;

    }
}